app/models/pregnancy.rb

```diff
@@ -71,7 +71,7 @@ def updating_pledge_sent?
   def pledge_info_presence
     errors.add(:pledge_sent, 'DCAF soft pledge field cannot be blank') if dcaf_soft_pledge.blank?
     errors.add(:pledge_sent, 'Patient name cannot be blank') if patient.name.blank?
-    errors.add(:pledge_sent, 'Clinic name cannot be blank') if patient.clinic_name.blank?
+    errors.add(:pledge_sent, 'Clinic name cannot be blank') if patient.clinic.blank?
```

Comment:
Just checked to be absolutely sure and it's looking like yes: 

```
> a  = Patient.first
=> #<Patient _id: 58e00ae2311c884c34bedf98, ...>

> a.clinic
=> nil

> a.clinic.blank?
=> true
```

Thanks for flagging this!